### DISCUSSION TITLE
Is there a way to customize the area of the "next sentence" click?

### DISCUSSION THREAD

**Discussion by GoldExperience** [dialogic-godot/dialogic#1722] [Q&A]
Is there a way to customize the scope of the "next sentence" click?
When I play a game, clicking anywhere in the game window triggers the next sentence, which causes it to conflict with other UI elements.
It would be much easier if the next sentence was only triggered when clicking on a text box.
I've tried looking at all the elements and scripts in the default layout, as well as all the settings in the style, and I can't find any code for this logic.

**Comment by Jowan-Spooner** [comment]
This has been a problem for some time and should be adressed before the 2.0 release. 
I've made an issue with the approach I'll most likely take (#1727).

**Reply by Jowan-Spooner** [reply]
So no, right now it's not possible to do that.
